space_age/app/views/admin/users/show.html.erb

```diff
@@ -16,38 +20,32 @@
         <%= f.submit "Update Account", class: "button expanded"%>
         <% end %>
       </div>
-    </div>
-    <div class="accordian-item" data-accordian-item>
-       <a href="#" class="accordian-title">Delete Account</a>
-       <div class="accordian-content" data-tab-content>
-         <h6 class="centered-text">Are You Sure? We'll Miss You So!</h6>
-         <%= button_to "I\'m Sure- Delete my Account", user_path(@user), method: :delete, class: "button expanded" %>
-       </div>
-     </div>
-    <div class="accordian-item" data-accordian-item>
-      <a href="#" class="accordian-title">Create a New Destination</a>
-      <div class="accordian-content" data-tab-content>
-        <%= form_for @destination do |f| %>
-        <%= f.text_field :planet, placeholder: "Enter a Planet, Nebula, or Region of Space" %>
-        <%= f.submit class: "button expanded"%>
-        <% end %>
```

Comment:
I'm glad we took this out for tommorow, but let's put it back- it will be SO easy to implement.